presto-mysql/src/test/java/com/facebook/presto/plugin/mysql/TestMySqlIntegrationSmokeTest.java

```diff
@@ -64,31 +68,34 @@ public final void destroy()
     public void testDescribeTable()
             throws Exception
     {
-        // we need specific implementation of this tests due to specific Presto<->Mysql varchar length mapping.
-        MaterializedResult actualColumns = computeActual("DESC ORDERS").toJdbcTypes();
-
-        // some connectors don't support dates, and some do not support parametrized varchars, so we check multiple options
+        execute("CREATE TABLE tpch.test (test_bigint bigint, test_varchar varchar(255), test_vardiff varchar(15), test_double double, test_date date, test_integer integer, test_timestamp timestamp)");
```

Comment:
Prefix column names with `c_` instead of `test_`. For instance `c_varchar_255`, `c_varchar_15`, `c_timestamp`, etc.